ts/Extensions/Annotations/Types/CrookedLine.ts

```diff
@@ -156,19 +156,6 @@ CrookedLine.prototype.defaultOptions = merge(
          * Additional options for an annotation with the type.
          */
         typeOptions: {
-            /**
-             * This number defines which xAxis the point is connected to.
-             * It refers to either the axis id or the index of the axis
-             * in the xAxis array.
-             */
-            xAxis: 0,
-            /**
-             * This number defines which yAxis the point is connected to.
-             * It refers to either the axis id or the index of the axis
-             * in the xAxis array.
-             */
-            yAxis: 0,
-
```

Comment:
I think `typeOptions` does support `xAxis` and `yAxis`: https://api.highcharts.com/highstock/annotations.crookedLine.typeOptions.xAxis